### ISSUE TITLE
Clicking tags seems to do nothing

### ISSUE THREAD

**Issue by codewithtyler** [dannagle/dannagle.github.io#1]
I am able to click on a tag at the top of the page but nothing happens. For example, in the 2023 session list, there are 3 DevOps related sessions. When I click on the tag the tag gets highlighted but as a user, I expect to see the list filter down to that tag but that never happens. I see the tag get highlighted and in the developer console, a message about needing to add or remove a highlight occurs.

**Comment by dannagle** [comment]
Thanks for this feedback. 

I actually was not sure what a user expects to see when clicking the tags. Highlighting them in the table was simply something I chose from a lack of other ideas. 

I'll flag this to tweak for next year's conference.

**Comment by codewithtyler** [comment]
Yeah no worries I've been using your schedule list a lot this year while running the live streams for the sessions and so I decided to click around to see if what some of the features did.
